openpathsampling/storage/snapshot_store.py

```diff
@@ -28,35 +58,14 @@ def _load(self, idx):
             the loaded snapshot instance
         """
 
-        s_idx = int(idx / 2) * 2
-
-        reversed_idx = 2 * s_idx + 1 - idx
         try:
-            obj = self.cache[reversed_idx]
-            snapshot = Snapshot(
-                configuration=obj.configuration,
-                momentum=obj.momentum,
-                is_reversed=not obj.is_reversed,
-                reversed_copy=LoaderProxy(self, reversed_idx)
-            )
-            return snapshot
-
-
+            return self._get(idx, True)
         except KeyError:
-            pass
-
-        configuration = self.vars['configuration'][s_idx]
-        momentum = self.vars['momentum'][s_idx]
-        momentum_reversed = self.vars['momentum_reversed'][idx]
+            return self._get(idx)
 
-        snapshot = Snapshot(
-            configuration=configuration,
-            momentum=momentum,
-            is_reversed=momentum_reversed,
-            reversed_copy=LoaderProxy(self, reversed_idx)
-        )
-
-        return snapshot
+    def _put(self, idx, snapshot):
+        self.vars['momentum_reversed'][idx] = snapshot.is_reversed
+        self.vars['momentum_reversed'][idx ^ 1] = not snapshot.is_reversed
```

Comment:
Hmmm, get and put. Load and save was taken. get and put is what is used in databases I think. Do you have a better name? This is the function to get or put the data from the storage.